app/jobs/foreman_dlm/refresh_dlmlock_status.rb

```diff
@@ -0,0 +1,17 @@
+module ForemanDlm
+  class RefreshDlmlockStatus < ApplicationJob
+    queue_as :refresh_dlmlock_status_queue
+
+    def perform(host_ids)
+      Host::Managed.where(id: host_ids).each(&:refresh_dlmlock_status)
+    end
+
+    rescue_from(StandardError) do |error|
+      Foreman::Logging.logger('background').error("Refresh Distributed Lock status: Error #{error}: #{error.backtrace}")
```

Comment:
This could use [Foreman::Logging.exception](https://github.com/theforeman/foreman/blob/7673103014073bf01908b5e7252956d5b35d4efa/lib/foreman/logging.rb#L82-L102).